cicd/3-app/javabuilder/config/test.config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Parameters": {
+    "SubDomainName": "javabuilder-test",
+    "BaseDomainName": "code.org",
+    "BaseDomainNameHostedZonedID": "Z2LCOI49SCXUGU",
+    "ProvisionedConcurrentExecutions": "1",
+    "ReservedConcurrentExecutions": "3",
+    "LimitPerHour": "10",
```

Comment:
As per [this thread](https://codedotorg.slack.com/archives/C01EF4GJ9GE/p1653689916931129), I think we'll need to bump this up to avoid getting request limit warnings in integration tests. Let's do 15?